pyproject.toml

```diff
@@ -129,6 +130,8 @@ lint.per-file-ignores."Tests/oss-fuzz/fuzz_font.py" = [
 lint.per-file-ignores."Tests/oss-fuzz/fuzz_pillow.py" = [
   "I002",
 ]
+lint.flake8-pytest-style.parametrize-names-type = "csv"
+
 lint.isort.known-first-party = [
```

Comment:
```suggestion
lint.flake8-pytest-style.parametrize-names-type = "csv"
lint.isort.known-first-party = [
```
Nitpick

Reply:
A linter forced the order I committed.

Reply:
I wasn't suggesting the order be changed, simply removing the blank line in the middle.